apps/emqx_bridge/src/emqx_bridge_v2.erl

```diff
@@ -390,11 +368,16 @@ uninstall_bridge_v2_helper(
     CreationOpts = emqx_resource:fetch_creation_opts(Config),
     ok = emqx_resource_buffer_worker_sup:stop_workers(BridgeV2Id, CreationOpts),
     ok = emqx_resource:clear_metrics(BridgeV2Id),
-    %% Deinstall from connector
-    ConnectorId = emqx_connector_resource:resource_id(
-        connector_type(BridgeV2Type), ConnectorName
-    ),
-    emqx_resource_manager:remove_channel(ConnectorId, BridgeV2Id).
+    case combine_connector_and_bridge_v2_config(BridgeV2Type, BridgeName, Config) of
+        {error, _} ->
+            ok;
+        _CombinedConfig ->
```

Comment:
Tiny nit (since we don't care about the combined config, this may sound more clear when reading this part):

```suggestion
    case validate_referenced_connectors(BridgeV2Type, ConnectorName, BridgeName) of
        {error, _} ->
            ok;
        ok ->
```

Reply:
I know...  It's just that there's this validate function whose purpose is to check the referenced connector.  Since we don't care about the combined config, it seems easier to understand the purpose of a `validate_referenced_connectors` call here rather than `combine_...` which we discard the result of.  :wink:

Reply:
Will fix at follow-up PR